.circleci/config.yml

```diff
@@ -98,37 +98,31 @@ jobs:
     docker:
       - image: ferrarimarco/github-changelog-generator:1.14.3
     steps:
-      - setup_remote_docker:
-          version: 18.06.0-ce
+      - setup_remote_docker
       # checkout code to default ~/project
       - checkout
       - run:
           name: Build changelog
           # $CHANGE_TOKEN is generated via the GitHub web UI, and then securely stored within CircleCI web UI
           command: |
             mkdir ~/changelog_build
-            git status
-            if (git log -1 --pretty=%s | grep Merge*) && (! git log -1 --pretty=%b | grep REL:) ; then
-              github_changelog_generator \
-                --exclude-tags-regex "^(?!v).*?$" \
-                --user bids-standard \
-                --project bids-specification \
-                --token ${CHANGE_TOKEN} \
-                --output ~/changelog_build/CHANGES.md \
-                --base ~/project/src/pregh-changes.md \
-                --header-label "# Changelog" \
-                --release-branch master \
-                --no-issues \
-                --no-filter-by-milestone \
-                --no-compare-link \
-                --pr-label "" \
-                --enhancement-label "" \
-                --bugs-label "" \
-                --exclude-labels "exclude-from-changelog"
-              cat ~/changelog_build/CHANGES.md
-            else
-              echo "Commit or Release, do nothing"
-            fi
+            github_changelog_generator \
+              --exclude-tags-regex "^(?!v).*?$" \
+              --user bids-standard \
+              --project bids-specification \
+              --token ${CHANGE_TOKEN} \
+              --output ~/changelog_build/CHANGES.md \
+              --base ~/project/src/pregh-changes.md \
+              --header-label "# Changelog" \
+              --release-branch master \
+              --no-issues \
+              --no-filter-by-milestone \
+              --no-compare-link \
+              --pr-label "" \
+              --enhancement-label "" \
+              --bugs-label "" \
+              --exclude-labels "exclude-from-changelog"
+            cat ~/changelog_build/CHANGES.md
```

Comment:
I think we could start running the changelog **generation** and **linting** on every commit to master ... and then commit this as previously, only on "Merge commits" to master.

This would allow us to earlier find issues with generation and linting.